cla_frontend/apps/zendesk/client.py

```diff
@@ -0,0 +1,81 @@
+import json
+import requests
+from requests.exceptions import ConnectionError, Timeout
+from django.conf import settings
+from django.template import loader, Context
+
+TICKETS_URL = 'https://ministryofjustice.zendesk.com/api/v2/tickets.json'
+ZENDESK_REQUESTER_ID = 762871298 # Civil Legal Advice
+ZENDESK_GROUP_ID = 24287107 # CLA Operator/Provider
+REQUEST_TIMEOUT = 10
+
+
+class ZendeskResponse(object):
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
```

Comment:
This is a bit hacky. Any reason we can't be explicit about what fields a ZendeskResponse object has? If not, do we need an object? Why not just use a dict?